### PR TITLE
Added Open Caveats in 2.11.2

### PR DESCRIPTION
**Pull request by selinaarokiaswamy-GSLAB**

### CODE REVIEW

**Comment by selinaarokiaswamy-GSLAB** [comment]
@JosephPJordan Could you please review and merge this pull request to staging branch

File: docs/release-notes.rst [hunk 1]
```diff
@@ -11,6 +11,17 @@ What's New in Version 2.11
 ------
 
 
+Open Caveats in v2.11.2
+===========================
+
+
+**1. APIC Endpoints in non-Common Route Domains are not dynamically detached from the BIG-IP pool until a manual sync is done**
+
+When the endpoints of an EPG are detached from the APIC, these endpoints are not dynamically removed/detached from the corresponding pool on the BIG-IP if these endpoints are in a non-Common Route Domain. A manual sync operation is required for the endpoints to be removed/detached from the BIG-IP.  This issue is not seen in Common Route Domain.
```
Comment (JosephPJordan) [inline docs/release-notes.rst:20]: For the last sentence, I suggest phrasing it something like: "This issue is not seen for route domains in the partition Common."
this is more similar to the BIG-IP documentation.

**Review by JosephPJordan — CHANGES_REQUESTED** [review]
One minor change

File: docs/release-notes.rst (hunk 1 above)
Comment (selinaarokiaswamy-GSLAB) [reply docs/release-notes.rst:20]: Hi @JosephPJordan. This statement "This issue is not seen for route domains in the partition Common." does not hold true for all the route domains in Common partition. How about "This issue is not seen for the partition default Route Domain in Common partition."

Comment (JosephPJordan) [reply docs/release-notes.rst]: Sounds good, but let's make it "... in the Common partition."

File: docs/release-notes.rst [hunk 2]
```diff
@@ -17,7 +17,7 @@ Open Caveats in v2.11.2
 
 **1. APIC Endpoints in non-Common Route Domains are not dynamically detached from the BIG-IP pool until a manual sync is done**
 
-When the endpoints of an EPG are detached from the APIC, these endpoints are not dynamically removed/detached from the corresponding pool on the BIG-IP if these endpoints are in a non-Common Route Domain. A manual sync operation is required for the endpoints to be removed/detached from the BIG-IP.  This issue is not seen in Common Route Domain.
+When the endpoints of an EPG are detached from the APIC, these endpoints are not dynamically removed/detached from the corresponding pool on the BIG-IP if these endpoints are in a non-Common Route Domain. A manual sync operation is required for the endpoints to be removed/detached from the BIG-IP. This issue is not seen for the partition default Route Domain in Common partition.
```
Comment (JosephPJordan) [inline docs/release-notes.rst:20]: Not sure if i lost my comment, but lets make it "...in the Common partition.'

Comment (selinaarokiaswamy-GSLAB) [reply docs/release-notes.rst]: Included this change

**Review by JosephPJordan — APPROVED** [review]

**Pull request merged**